composable_kernel/include/utility/dynamic_buffer.hpp

```diff
@@ -10,11 +10,16 @@ struct DynamicBuffer
 {
     using type = T;
 
+    using rawType = typename std::remove_const<T>::type;
```

Comment:
Please limit the usage of std as much as possible,

use ck::remove_cv_t instead of std::remove_const